README.md

```diff
@@ -38,7 +38,7 @@ brew install colima
 
 Before running your devcontainer, make sure colima is started.
 ```bash
-colima start
+colima start --cpu 4 --memory 8 --vz-rosetta --vm-type=vz
```

Comment:
fyi @s-amann this alone should largely remediate macOS build issues.

Reply:
👏 thank you

Reply:
if anyone had already ran `colima start` you may need to run `colima stop; colima delete` before attempting to run with these new vm options